sr/robot3/robot.py

```diff
@@ -70,6 +72,7 @@ def __init__(
         else:
             self._lock = obtain_lock()
         self._metadata: Optional[Metadata] = None
+        self._no_pb = no_powerboard
```

Comment:
Nit: I'm not a huge fan of calling this "pb". Perhaps reflect the argument and name it `_no_powerboard`?